src/kem/sike/external/P434/ARM64/fp_arm64_asm.S

```diff
@@ -4,6 +4,13 @@
 // Abstract: field arithmetic in 64-bit ARMv8 assembly for P434 on Linux
 //*******************************************************************************************
 
+// Format function and variable names for Mac OS X
+#if defined(__APPLE__)
+    #define fmt(f)    _##f
```

Comment:
Needs oqs namespacing as well. Compare with  liboqs/src/kem/sike/external/P434/AMD64/fp_x64_asm.S.

Reply:
Thanks. Don't know why I didn't put the namespacing in the first place! (need more coffee...)